### PR TITLE
Re-added kind-projector compiler plugin to backend.

### PR DESCRIPTION
**Pull request by kshakir**
Custom scaladoc settings must be appended (++=) instead of overwriting (:=).
Fixed unmoored doc warnings/errors.

### CODE REVIEW

File: backend/src/main/scala/cromwell/backend/BackendWorkflowInitializationActor.scala [hunk 1]
```diff
@@ -79,7 +79,7 @@ object BackendWorkflowInitializationActor {
       //This map append will overwrite default key/values with runtime settings upon key collisions
       val lookups = defaultRuntimeAttributes.mapValues(_.asWomExpression) ++ runtimeAttributes
 
-      runtimeAttributeValidators.toList.traverse[WishKindProjectorWorked, Unit]{
```
Comment (mcovarr) [inline backend/src/main/scala/cromwell/backend/BackendWorkflowInitializationActor.scala]: 😢

File: project/Dependencies.scala [hunk 2]
```diff
@@ -47,6 +48,7 @@ object Dependencies {
   val mouseV = "0.10-MF"
   val mysqlV = "5.1.42"
   val nettyHandlerV = "4.0.51.Final"
+  val paradiseV = "2.1.0"
```
Comment (cjllanwarne) [inline project/Dependencies.scala]: ToL `paradise 2.1` sounds like it's straight out of a horror/dystopia movie

**Comment by kshakir** [comment]
> Bonus points if you can ELI5?

Perhaps after I finish the FP book club??

At the moment, this is just fixing an sbt bug that was blocking [this commit](https://github.com/broadinstitute/cromwell/pull/2954/commits/8ce921581169f8466b67a6f03ceee8b498e4b483).
